.github/workflows/publish-package.yml

```
@@ -9,7 +9,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3
+    - name: Get full repo history because versioning
```

```suggestion
    - name: Get full repo history
```

I hate this suggestion, so feel free to ignore. But..._puts microsoft hat on_, since we're getting closer to v1 😢